src/metabase/query_processor/middleware/cache_backend/db.clj

```diff
@@ -14,15 +14,17 @@
   (when-let [{:keys [results updated_at]} (db/select-one [QueryCache :results :updated_at]
                                             :query_hash query-hash
                                             :updated_at [:>= (u/->Timestamp (- (System/currentTimeMillis)
-                                                                               (* 1000 max-age-seconds)))])]
+                                                                               (* 1000 max-age-seconds))
+                                                                            "UTC")])]
     (assoc results :updated_at updated_at)))
 
 (defn- purge-old-cache-entries!
   "Delete any cache entries that are older than the global max age `max-cache-entry-age-seconds` (currently 3 months)."
   []
   (db/simple-delete! QueryCache
     :updated_at [:<= (u/->Timestamp (- (System/currentTimeMillis)
-                                       (* 1000 (public-settings/query-caching-max-ttl))))]))
+                                       (* 1000 (public-settings/query-caching-max-ttl)))
+                                    "UTC")]))
```

Comment:
Does this assume that the server is always running in UTC?

Reply:
The UTC here is redundant. Since it's using `System/currentTimeMillis`, that function returns a millis since epoch in UTC. Since it's a long, timezone doesn't matter in that `->Timestamp` function. This query will be against the application database and I haven't spent much time looking into our application database and the queries against it. I think it should handle the parameters correctly, but would be good to confirm with a test.